src/AstGen.zig

```diff
@@ -1722,6 +1722,54 @@ fn structInitExpr(
         }
     }
 
+    {
+        var duplicate_names = std.AutoArrayHashMap(u32, ArrayListUnmanaged(Ast.TokenIndex)).init(astgen.arena);
+        defer duplicate_names.deinit();
+        try duplicate_names.ensureTotalCapacity(@intCast(struct_init.ast.fields.len));
```

Comment:
Oh, looks like you haven't pushed the variant using SFBA - make sure to push that!